apps/angular/projection/src/app/ui/list-item/list-item.component.ts

```diff
@@ -1,35 +1,24 @@
-import { Component, Input } from '@angular/core';
-import { StudentStore } from '../../data-access/student.store';
-import { TeacherStore } from '../../data-access/teacher.store';
-import { CardType } from '../../model/card.model';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
 
 @Component({
   selector: 'app-list-item',
   template: `
     <div class="border-grey-300 flex justify-between border px-2 py-1">
-      {{ name }}
-      <button (click)="delete(id)">
+      <ng-content></ng-content>
+      <button (click)="delete(item.id)">
```

Comment:
you don't need to pass the item. delete can be a simple event emitter.

Reply:
done

Reply:
done